frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/image/embed_image_url_widget.dart

```diff
@@ -1,7 +1,15 @@
+import 'package:flutter/material.dart';
+
 import 'package:appflowy/generated/locale_keys.g.dart';
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flowy_infra_ui/flowy_infra_ui.dart';
-import 'package:flutter/material.dart';
+
+/// This pattern allows for both HTTP and HTTPS Scheme
+/// It allows for query parameters
+/// It only allows the following image extensions: .png, .jpg, .gif, .webm
+///
+const String _imgUrlPattern =
```

Comment:
I know this isn't directly related to this PR, but it reminds me that we should create a pattern collection in the shared folder. This way, we can share all the current patterns and avoid having to define them repeatedly.

Reply:
Done :+1: